### PR TITLE
Added Intro, removed TODO

### PR DESCRIPTION
**Pull request by OzBob**
Updated Description using text from Mark's article
Made 'UPDATE THIS' notes non-public comments for editors only - I'm still learning MD so I hope that syntax works ...

## Summary

removed TODO and UPDATETHIS

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by ThatRendle — APPROVED** [review]

File: docs/architecture/grpc-for-wcf-developers/index.md [hunk 1]
```diff
@@ -45,21 +45,21 @@ Editors:
 
 ## Introduction
 
-TODO
+Helping machines communicate with each other has been one of the primary preoccupations of the digital age.
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/index.md]: Maybe a simpler language?

```suggestion
Helping machines communicate with each other has been one of the main concerns of the digital age. 
```

File: docs/architecture/grpc-for-wcf-developers/index.md [hunk 2]
```diff
@@ -45,21 +45,21 @@ Editors:
 
 ## Introduction
 
-TODO
+Helping machines communicate with each other has been one of the primary preoccupations of the digital age. 
 
 ## Purpose
 
-TODO
+The popular framework 'gRPC' is a natural choice for new apps and WCF apps moving to .NET Core.  This is a short introduction to building gRPC Services in ASP.NET Core 3.0 and offers guidance for migrating an existing WCF application to gRPC.
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/index.md]:
```suggestion
The popular framework gRPC is a natural choice for new apps and existing WCF apps moving to .NET Core.  This book is a short introduction to building gRPC services in ASP.NET Core and offers guidance for migrating an existing WCF application to gRPC.
```

File: docs/architecture/grpc-for-wcf-developers/index.md [hunk 3]
```diff
@@ -45,21 +45,21 @@ Editors:
...
 
 ## Who should use this guide
 
-**UPDATE THIS**
+[//]: #(UPDATE THIS)
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/index.md]: Should we just remove this?

```suggestion
```

File: docs/architecture/grpc-for-wcf-developers/index.md [hunk 4]
```diff
@@ -45,21 +45,21 @@ Editors:
...
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/index.md]:
```suggestion
```

File: docs/architecture/grpc-for-wcf-developers/index.md [hunk 5]
```diff
@@ -45,21 +45,21 @@ Editors:
...
 
 The audience for this guide is WCF developers, development leads, and architects who are interested in migrating WCF solutions on .NET 4 and earlier to ASP.NET Core 3.0 using gRPC services.
 
 ## How you can use this guide
 
-**UPDATE THIS**
+[//]: #(UPDATE THIS)
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/index.md]:
```suggestion
```

File: docs/architecture/grpc-for-wcf-developers/index.md [hunk 6]
```diff
@@ -45,21 +45,21 @@ Editors:
...
```
Comment (mairaw) [inline docs/architecture/grpc-for-wcf-developers/index.md]:
```suggestion
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @OzBob. Left some suggestions.

File: docs/architecture/grpc-for-wcf-developers/index.md (hunk 3 above)
Comment (mairaw) [reply docs/architecture/grpc-for-wcf-developers/index.md]: Still unclear why these are still here.

**Comment by mairaw** [comment]
@OzBob can you please address my questions? @markrendle @shirhatti what was the plan for these two sections?

**Comment by nishanil** [comment]
I emailed @markrendle. He is planning to address it this weekend.

**Comment by mvelosop** [comment]
Closing this PR as it's beeing replaced with https://github.com/dotnet/docs/pull/15586

**Pull request closed without merging**